torch/autograd/gradcheck.py

```diff
@@ -207,7 +207,8 @@ def gradcheck(
     rtol: float = 1e-3,
     raise_exception: bool = True,
     check_sparse_nnz: bool = False,
-    nondet_tol: float = 0.0
+    nondet_tol: float = 0.0,
+    check_undefined_grad: bool = True
```

Comment:
Can you add the same flag to the `gradgradcheck` function below and forward it when `gradcheck` is called so that people can override it in this case as well.